python/paddle/tensor/search.py

```diff
@@ -765,3 +765,75 @@ def topk(x, k, axis=None, largest=True, sorted=True, name=None):
         attrs=attrs)
     indices.stop_gradient = True
     return values, indices
+
+
+def searchsorted(sorted_sequence,
+                 values,
+                 out_int32=False,
+                 right=False,
+                 name=None):
+    """
+    This OP is used to find the index of the corresponding `sorted_sequence` in the innermost dimension based on the given `values`.
+
+    Args:
+        sorted_sequence(Tensor): An input N-D or 1-D tensor with type int32, int64, float32, float64. The value of the tensor monotonically increases in the innermost dimension. 
+        values(Tensor or Scalar): An input N-D tensor or a Scalar value with type int32, int64, float32, float64.
+        out_int32(bool, optional): Data type of the output tensor which can be int32, int64. The default value is False, and it indicates that the output data type is int64.
+        right(bool, optional): Find the upper and lower bounds of the sorted_sequence range in the innermost dimension based on the given `values`. If the sorted_sequence value is nan or inf, return the size of the innermost dimension
```

Comment:
`upper and lower bounds` -> `upper or lower bounds`.

`return the size of the innermost dimension` -> `return the size of the innermost dimension.`